common/src/main/java/com/wynntils/wynn/event/SpellCastedEvent.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.wynn.event;
+
+import com.wynntils.wynn.objects.SpellType;
+import net.minecraftforge.eventbus.api.Event;
+
+public class SpellCastedEvent extends Event {
```

Comment:
Just a grammar note. Is not the paste tense form of "cast" still "cast"? So it should be `SpellCastEvent`?

Reply:
Yep looks like it. I have no idea where I came up with the word "casted"